content/departments/cloud/technical-docs/v1.1/mi1-1_creation_process.md

```diff
@@ -127,6 +134,26 @@ terraform output
 
 Provide the value of `cloud_nat_ips` to CE or customers, and instruct them to allow incoming traffic from referenced IP addresses.
 
+### Giving customer access
+
+> NOTE: ⚠️ Before providing access to the customer, make sure that the GCP alerting policy has been created!
+
+#cloud usually hands off to CE at this point, they will schedule a call with the customer (including a DevOps team member, if needed) to walk the site admin on the customer's side through performing initial setup of the product including adding the license key, adding repos, configuring SSO, and inviting users. Please notify the CE requested the instance has been created with the following message.
+
+```
+Hi,
+
+The instance is ready. Would you kindly add 10 extra seats to the license so we can have a few extra seats for Sourcegraph management access?
+
+Here's the link to the password reset link <>. Please note the link will expire after 24 hours
```

Comment:
@michaellzc as we have sharing password via 1pass, this reset link is 1pass link? b/c we cannot share passwords in such messages anymore :)